parser/expandpath_test.go

```diff
@@ -0,0 +1,57 @@
+package parser
+
+import (
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+)
+
+func mockCurrentUser() (*user.User, error) {
+	return &user.User{
+		Username: "testuser",
+		HomeDir:  "/home/testuser",
+	}, nil
+}
+
+func mockLookupUser(username string) (*user.User, error) {
```

Comment:
Same as above, this is used in one place so it might be simpler to just use it in the test below